CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 3.0)
 
-if (NOT CMAKE_VERSION VERSION_LESS 3.12)
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.13)
```

Comment:
Sorry for an unsolicited comment, but I thought this could help here. I'm usually doing just the following -- that way I don't need to look up what version the policy appeared in and so it's less error-prone:

```cmake
if(POLICY CMP0077)
    cmake_policy(SET CMP0077 NEW)
endif()
```

Reply:
oh, great, thats even better!

thanks!